### PR TITLE
Add openSUSE_Leap_15-x86_64 target

### PR DESCRIPTION
**Pull request by niner**

### CODE REVIEW

**Comment by niner** [comment]
openSUSE provides two distinct distributions: openSUSE Leap (the traditional one) and openSUSE Tumbleweed (the rolling release one).

We need wkhtmltopdf on openSUSE Leap 15.0 and will upgrade our servers to 15.1 at a later date. That's why I did it for 15.0 this time.

**Comment by ashkulz** [comment]
> openSUSE provides two distinct distributions: openSUSE Leap (the traditional one) and openSUSE Tumbleweed (the rolling release one).

Makes sense. Can you change the target name to `opensuse-leap-15-x86_64`?

> We need wkhtmltopdf on openSUSE Leap 15.0 and will upgrade our servers to 15.1 at a later date. That's why I did it for 15.0 this time.

I think the PR should use `15`, while you can always use the locally build RPMs for your servers.

**Comment by ashkulz** [comment]
@niner: would you be able to make the changes above?

File: build.yml [hunk 1]
```diff
@@ -542,5 +563,5 @@ fpm-params:
   vendor:       wkhtmltopdf
   maintainer:   Ashish Kulkarni <ashish@kulkarni.dev>
   url:          https://wkhtmltopdf.org/
-  prefix:       /usr/local
+  prefix:       /usr
```
Comment (ashkulz) [inline build.yml]: This affects every target, and I don't think we should ever install in `/usr`.

**Comment by ashkulz** [comment]
As I don't personally use this distribution, I'm hesitant to merge something which I won't be testing. If you do find the time to come back to this in the future, feel free to reopen the PR and I'll review it again 👍

**Comment by ashkulz** [comment]
Thanks for the contribution, @niner! I did use it when implementing e5bd52b3faa8408b7f5a92fa95c6ebfdece15f74 👍

**Comment by niner** [comment]
Sorry for not replying sooner. I have been swamped with work. Thanks for picking this up!

**Comment by ashkulz** [comment]
No worries, I just took what you did and tweaked it a bit. We now have [3 openSUSE packages for 0.12.6](https://github.com/wkhtmltopdf/packaging/releases/latest) due to that :+1:

**Pull request closed without merging**